fusion-plugin-rpc/src/browser.js

```diff
@@ -41,11 +48,17 @@ class RPC {
       throw new Error('Missing emitter registered to UniversalEventsToken');
     }
     const fetch = this.fetch;
+    const config = this.config;
     const emitter = this.emitter;
+
+    let apiPath: string = 'api';
```

Comment:
This logic could be moved to the constructor, rather than each request, especially if we add some logic to remove duplicate `/`